demo.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="assets/css/styles.css">
+  <script src="assets/scripts/script.js" defer></script>

```

Comment:
Move JS linking script tag to the last line within your Body tags. It allows the html and styles to load on the page before all of the JS which could cause lag/delays with loading

Reply:
as mentioned in a previous comment, the 'defer' attribute does this and allows to keep all externally linked files in the same place for organisations sake. 

more info may be found here: https://www.w3schools.com/tags/att_script_defer.asp#snhb-main_leaderboard-0:~:text=When%20present%2C%20it%20specifies%20that%20the%20script%20is%20executed%20when%20the%20page%20has%20finished%20parsing.